source/style-guide/style/code-examples.txt

```diff
@@ -10,27 +10,12 @@ Code Examples
    :depth: 1
    :class: singlecol
 
-Use the following guidelines when creating blocks of code as input
-or output examples:
+Use the following guidelines when creating blocks of code:
 
 -  Don't use screenshots to show code examples. Format them as blocks
    of code by using the appropriate markup in your authoring tool. For
    more information about formatting, see :ref:`text-formatting`.
 
--  When showing input, always include a command prompt (such as $).
-
--  As often as necessary, show input and output in separate blocks and
-   provide explanations for each. For example, if the input contains
-   arguments or parameters, explain those. If the user should expect
-   something specific in the output, or you want to show only part of
-   lengthy output, provide an explanation.
-
--  When the command is simple, and there's nothing specific to say
-   about the output, you can show the input and output in the same code
-   block, as users would actually see the code in their own terminal.
-   The inclusion of the command prompt differentiates the input from
-   the output.
-
 -  Ensure that any placeholder text in code is obvious.
 
    - If the authoring tool allows it, apply italics to placeholders; if
```

Comment:
Re: "If the authoring tool allows it, apply italics to placeholders..."

Do any of our authoring tools allow us to use italics in code blocks?